### DISCUSSION TITLE
Lesson 7 Part 19 13:13 Getting error "now owner" at the test.

### DISCUSSION THREAD

**Discussion by BowtiedHarpyEagle** [Cyfrin/foundry-full-course-cu#955] [Q&A] [answered]
I'm going through the lesson 7 and I feel totally lost around the second half of part 19, with the integration tests. 

Right now I'm getting "not owner" error when I run the test in 13:13 timestamp. 

https://youtu.be/5u02NBfV4PY?list=PL2-Nvp2Kn0FPH2xU3IbKrrkae-VVXs1vk&t=793

The test at 11:29 error message also was slightly different. My terminal did not mention anything about prank-broadcasting, just that the test failed. 

This is my second time going through this lesson 7. So this time I committed every small change through the course I made to the code starting with the beginning of lesson 7. Hoping this way I can isolate and pinpoint at what point I made what mistake. The whole repo with the full commit history is on my github here: 

https://github.com/BowtiedHarpyEagle/foundry-fund-me-f23

I'm also adding a screenshot of the latest failed test message.

![screen](https://github.com/Cyfrin/foundry-full-course-f23/assets/140409172/0a35f24b-492a-400d-91e9-d4048dff2e6f)

Please help. I'm questioning my ability to learn web3 security, feeling I should maybe give up.

Now I'll take a break...

**Comment by 0xusmanf** [comment, marked as answer]
Please add `vm.startBroadcast();` before and `vm.stopBroadcast();` after [this](https://github.com/BowtiedHarpyEagle/foundry-fund-me-f23/blob/6fe075b9de0f0d518b925afcbcca5e16bce71fe2/script/Interactions.s.sol#L35) line in your code.

And don't worry even the best of the best goes through situations like this. Keep it up 👍🏻, we all are here to help you. Just let us know if you hit any bumps on the way.

LFG 🚀 🚀 🚀 🔥 🔥 🔥

**Reply by BowtiedHarpyEagle** [reply]
Thanks for both the kind words and the code correction. It worked, and it is such a huge relief.

**Reply by 0xusmanf** [reply]
Always happy to help.

**Reply by stack1171** [reply]
@usmanfarooq91 thank you for that, I was troubleshooting the issue for a while....
